halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/deploy/DiffDeployCommand.java

```diff
@@ -20,23 +20,23 @@
 import com.beust.jcommander.Parameters;
 import com.netflix.spinnaker.halyard.cli.command.v1.NestableCommand;
 import com.netflix.spinnaker.halyard.cli.services.v1.Daemon;
+import com.netflix.spinnaker.halyard.cli.ui.v1.AnsiFormatUtils;
 import com.netflix.spinnaker.halyard.cli.ui.v1.AnsiUi;
 import lombok.AccessLevel;
 import lombok.Getter;
 
 @Parameters()
-public class PlanDeployCommand extends NestableCommand {
+public class DiffDeployCommand extends NestableCommand {
   @Getter(AccessLevel.PUBLIC)
-  private String commandName = "plan";
+  private String commandName = "diff";
 
   @Getter(AccessLevel.PUBLIC)
-  private String description = "This is effectively a dry-run of your Spinnaker deployment, displaying what changes"
-      + " will be made, and how your deployment will look.";
+  private String description = "This shows what changes you have made since Spinnaker was last deployed.";
```

Comment:
Good idea - there's a few things missing here that I still need to add (e.g. which versions are being updated, did any dependent files change, etc...) I'll add that to the list.